packages/ckeditor5-list/src/todolistediting.js

```diff
@@ -106,10 +105,19 @@ export default class TodoListEditing extends Plugin {
 		// <blockquote><p>Foo{}</p></blockquote>
 		// <ul><li><checkbox/>Bar</li></ul>
 		//
-		// Note: When content language direction is RTL, the behaviour is mirrored.
-		const localizedJumpOverCheckmarkKey = editor.locale.contentLanguageDirection === 'ltr' ? 'arrowleft' : 'arrowright';
+		this.listenTo( editing.view.document, 'keydown', ( eventInfo, domEventData ) => {
+			const direction = getLocalizedArrowKeyCodeDirection( domEventData.keyCode, editor.locale.contentLanguageDirection );
 
-		editor.keystrokes.set( localizedJumpOverCheckmarkKey, ( evt, stop ) => jumpOverCheckmarkOnSideArrowKeyPress( stop, model ) );
+			if ( direction != 'left' ) {
+				return;
+			}
+
+			if ( jumpOverCheckmarkOnSideArrowKeyPress( model ) ) {
+				domEventData.preventDefault();
+				domEventData.stopPropagation();
+				eventInfo.stop();
+			}
```

Comment:
Why not keep that in `jumpOverCheckmarkOnSideArrowKeyPress()` together:

```javascript
this.listenTo( editing.view.document, 'keydown', getJumpOverCheckmarkOnSideArrowKeyPress( editor ) );
// or just editor.locale.contentLanguageDirection + editor.model
```

so, no returning Boolean and stopping event propagation outside.

Reply:
Yeah. Especially since now the method returns:

1.  `undefined`
2.  `true`
3.  `false`

There's something strange and at least should be double checked.